libs/blocks/merch/merch.js

```diff
@@ -374,7 +374,7 @@ export async function openModal(e, url, offerType) {
   if (/\/fragments\//.test(url)) {
     const fragmentPath = url.split(/hlx.(page|live)/).pop();
     modal = await openFragmentModal(fragmentPath, getModal);
-  } else if (/^https?:/.test(url)) {
```

Comment:
can we rather try `/^(https)?\//` ? this way it's either https (original requirement) or relative path?

Reply:
ok if we neither wants https://... URLs i'm fine thought relative paths were an option